packfile.c

```diff
@@ -352,6 +352,34 @@ void close_all_packs(struct raw_object_store *o)
 	}
 }
 
+void unlink_pack(const char *pack_name, int ignore_keep)
+{
+	const char *exts[] = {".pack", ".idx", ".keep", ".bitmap", ".promisor"};
```

Comment:
nit: if you make this "static const char *exts[] ....", it initializes at compile time and saves 10 instructions...